src/full/Agda/Compiler/MAlonzo/Compiler.hs

```diff
@@ -555,17 +555,15 @@ term tm0 = mkIf tm0 >>= \ tm0 -> do
       erased  <- lift $ getErasedConArgs c
       let missing = drop (length ts) erased
           notErased = not
-      case all notErased missing of
+      if all notErased missing then (do
```

Comment:
Avoid parenthesis (this might require replacing the `if/then/else` with a multiway `if`)